views/overlays/admin.hbs

```diff
@@ -185,6 +211,48 @@
 		toastNotify();
 	}
 
+	function sendChallongeUpdate() {
+		var data = {
+			'challongeUrl': document.getElementById('challongeUrl').value,
+		};
+		challongeSocket.emit('update challonge', data);
+		toastNotify();
+	}
+
+	function createMatchList(data) {
+		var matchList = $('#challongeMatchList');
+		if (matchList) {
+			matchList.remove();
+		}
+
+		var selectList = document.createElement('select');
+		selectList.id = 'challongeMatchList';
```

Comment:
The server will now report if the matches were actually updated so this method will only fire when necessary.